debian/rules

```diff
@@ -22,3 +22,6 @@ override_dh_auto_install:
 	dh_install build/cmd/* /usr/bin
 	dh_install lib/* /usr/share/performance-diagnostics/lib
 	dh_install bpf/* /usr/share/performance-diagnostics/bpf
+	dh_install telegraf/delphix-telegraf-service telegraf/perf_playbook /opt/delphix/server/bin
```

Comment:
I would try not to tie yourself to delphix appstack paths like `/opt/delphix`. You can just install in `/usr/bin`

Reply:
Done